### PR TITLE
fix(InputManager): Fix hide/restore event fd's

### PR DESCRIPTION
**Pull request by pastaq**
- Use hidden path to process events.
- Support getting event from /dev/input and /dev/input/.hidden
- Reset logger to INFO on all modules.
- Restore input on clean exit.
- Restore input on startup if crashed.

### CODE REVIEW

File: docs/plugins/TUTORIALS.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ extends Library
 # Called when the node enters the scene tree for the first time.
 func _ready() -> void:
         super()
-        logger = Log.get_logger("vkCube", Log.LEVEL.DEBUG)
+        logger = Log.get_logger("vkCube", Log.LEVEL.INFO)
```
Comment (ShadowApex) [inline docs/plugins/TUTORIALS.md:40]: lol

File: core/global/input_manager.gd [hunk 2]
```diff
@@ -185,6 +195,7 @@ func _on_gamepad_change(_device: int, _connected: bool) -> void:
 	if handheld_gamepad and not handheld_gamepad.is_open():
 		if handheld_gamepad.open() != OK:
 			logger.error("Unable to open handheld keyboard device.")
+		hide_event_device(handheld_gamepad.kb_event_path)
```
Comment (ShadowApex) [inline core/global/input_manager.gd:198]: So are we only hiding handheld gamepads? Should we also be hiding regular gamepads..?

**Review by ShadowApex — APPROVED** [review]

File: core/global/input_manager.gd (hunk 2 above)
Comment (pastaq) [reply core/global/input_manager.gd:198]: Regular gamepads get hidden a few lines above. IDK why it's not showing in the diff

https://github.com/ShadowBlip/OpenGamepadUI/blob/60e1733230f57ca499de256cc72dbbf6926f6614/core/global/input_manager.gd#L193

File: docs/plugins/TUTORIALS.md (hunk 1 above)
Comment (pastaq) [reply docs/plugins/TUTORIALS.md:40]: sed is thorough

**Pull request merged**